### DISCUSSION TITLE
Scroll layers without isolating

### DISCUSSION THREAD

**Discussion by Bataghast** [wojtryb/Shortcut-Composer#94] [Ideas]
I love the layer scroll feature, but sometimes the isolate transparency grid can be visually disorienting. it would be helpful to have an option to turn off the isolate component of the layer scroll for users who prefer the layer stack for visual feedback.
even better if it's a toggle that can be added to one of the pie menus!

thank you for all your hard work on this plugin!

**Comment by wojtryb** [comment]
Hi,

Currently I do not plan to support this kind of customization. The best you can do is to navigate to the action code:

`Settings > Manage Resources > Open Resource Folder > pykrita > shortcut_composer > actions.py`

and then remove the instruction to isolate the layer in this part of code:
```python
    templates.CursorTracker(
        name="Scroll isolated layers",
        instructions=[instructions.TemporaryOn(Toggle.ISOLATE_LAYER)],
        vertical_slider=Slider(
            controller=controllers.ActiveLayerController(),
            values=CurrentLayerStack(PickStrategy.ALL),
        ),
    ),
```

You just need to change the line
`instructions=[instructions.TemporaryOn(Toggle.ISOLATE_LAYER)],`
to:
`instructions=[],`

Sadly this will not let you to change it back and forth on the go.

**Reply by Bataghast** [reply]
thank you! this solution works for me.
